KEYHOLDER.md

```diff
@@ -1,40 +1,84 @@
+## Outline
 
-```
+Keyholders MUST subscribe to the [Sigstore Maintainer Calendar](https://calendar.google.com/calendar/u/0?cid=Y19ydjIxcDJuMzJsbmJoYW5uaXFwOXIzNTJtb0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t) for upcoming root signing event. Root signing events are expected to occur about every 4-5 months. The next `v+1` version signing will be scheduled, and the `v+2` version will be tentatively scheduled. Testing will occur the week before the signing. Keyholders are required to communicate that they have completed the [pre-work](#signing-pre-work) to the orchestrator through [#sigstore-keyholder](https://sigstore.slack.com/archives/C03E4HP6RCK) Slack channel. All testing and signing events can occur asynchronously. Keyholders are expected to be "on-call" during the root signing window in case there is an issue.
+
+### Pre-requisites
+
+Ensure you have the following:
+- [ ] A local Git installation and a Go development setup
+- [ ] SSH authentication for GitHub (see [here](https://docs.github.com/en/authentication/connecting-to-github-with-ssh))
+- [ ] A USB port connection for your hardware key (beware of using a remote connection; the keyholder should not assume that magic occurs during an SSH session)
+
+### Signing pre-work
+
+During a root signing test, keyholders must complete the following steps in order:
+- [ ] Fork the [root-signing](https://github.com/sigstore/root-signing) repository by clicking the "fork" button on GitHub. 
+- [ ] Test binary build: Set your `${GITHUB_USER}` with your GitHub username and execute the script:
+```bash
 export GITHUB_USER=${YOUR_GITHUB_USER}
 ./scripts/step-0.sh
 ```
-
-The conductor should also set the online snapshot and timestamp keys, and the previous repository if it exists. The key references should have the a go-cloud style URI like `gcpkms://<some key>`.
-
+This will setup your fork and build the TUF binary to use for metadata generation.
+- [ ] (If you are a new keyholder) Test registering your new key: Do not use a key that you need -- this will wipe the key! Set the following environment variables, and then follow the steps in [Registering a new root key](#registering-a-new-root-key)
+```bash
+export LOCAL=1
+```
+
+**CONFIRM** that you created a new directory under `ceremony/$DATE/keys/` with a new serial numbered `XXXXXX` directory. Run 
+```bash
+./scripts/verify.sh
+```
+and confirm that there is some output with `VERIFIED KEY WITH SERIAL NUMBER XXXXXX`.
+
+- [ ] Test signing: Note you will need a test GCP signer. Sigstore keyholders have access to the test KMS key below. You will need to authenticate with GCP. Run the following. 
+```bash
+export LOCAL=1
+gcloud auth application-default login
+export TEST_KEYS=gcpkms://projects/project-rekor/locations/global/keyRings/sigstore-root/cryptoKeys
```

Comment:
Isn't this the prod keys? 

Test keys should live in the staging project to avoid crossover.

Reply:
It's a test key I had set up in that key ring.

Sure, I can make a staging project test key.

Reply:
Where is the staging project? I'm not sure I have access

Reply:
Projectsigstore-staging, if you don’t have access, we can add you, I’d ask Simon, not sure what’s the recommend way now since we moved the project.

Reply:
done! updated the key